stories/Radio.stories.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+import { Radio as Component } from "../confetti-ds/src";
+
+export default {
+  title: "Radio/Radio",
+  component: Component,
+  argTypes: {
+    value: {
+      control: {
+        type: "select",
+        options: ["string", "number", "bool"]
+      }
+    }
+  }
+};
+
+export const Radio = (args) => (
+  <fieldset>
```

Comment:
oh, wait, I approved, but you forgot to add a `<legend>` with a label here.

Reply:
fixed